DuoUniversal/JwtUtils.cs

```diff
@@ -226,10 +226,15 @@ private static SigningCredentials GenerateSigningCreds(string secret)
         /// Generate a SecurityKey for the given shared secret 
         /// </summary>
         /// <param name="secret">The shared secret</param>
-        /// <returns>A SecurityKey encoding thae share secret</returns>
+        /// <returns>A SecurityKey encoding the shared secret</returns>
         private static SecurityKey GenerateSecurityKey(string secret)
         {
             byte[] keyBytes = Encoding.UTF8.GetBytes(secret);
+            // In case anything wants to enforce the correct key size, pad out to 64 bytes
+            if (keyBytes.Length < 64)
+            {
+                Array.Resize(ref keyBytes, 64);
```

Comment:
Before anyone asks.  Resize works by creating a new array of the target size.  The default value for a byte array is zero.  Then it copies over the old array starting from index 0.  So this _effectively_ right pads the array with zeroes, to the target size.

This is what the HMAC spec/algorithm calls for anyway, so I'm just doing it pre-emptively.